src/pages/patientView/PatientViewPageUtils.tsx

```diff
@@ -0,0 +1,19 @@
+import SampleManager from "./SampleManager";
+import { PatientViewPageStore } from "./clinicalInformation/PatientViewPageStore";
+import TumorColumnFormatter from "./mutation/column/TumorColumnFormatter";
+import { ClinicalDataBySampleId } from "shared/api/api-types-extended";
+import _ from "lodash";
+
+export function checkNonProfiledGenesExist( sampleIds:string[],
+                                            entrezGeneIds:number[],
+                                            sampleToGenePanelId:{[sampleId:string]:string},
+                                            genePanelIdToEntrezGeneIds:{[genePanelId:string]:number[]}
+):boolean {
+    return !! _.find(entrezGeneIds, entrezGeneId => {
```

Comment:
_.some is a little easier to read and returns boolean

Reply:
Good point. Updated code.